docs/source/kit/pi.rst

```diff
@@ -0,0 +1,25 @@
+Raspberry Pi
+============
+
+.. DANGER:: Do not reflash or edit the SD card. This will stop your robot working!
+
+.. figure:: /_static/kit/pi2.jpg
```

Comment:
There is both a `pi.jpg`, used in `docs/source/kit/index.rst`, and a `pi2.jpg`, used here. Do we really need both?

Reply:
`pi2.jpg` is a much better image. I suggest removing `pi.jpg`

Reply:
Added to #18.